openmetadata-service/src/main/java/org/openmetadata/service/resources/databases/DatabaseSchemaResource.java

```diff
@@ -644,6 +645,41 @@ public DatabaseSchema deleteDataProfilerConfig(
     return addHref(uriInfo, databaseSchema);
   }
 
+  @GET
+  @Path("/getEntityRelationship")
```

Comment:
lets not add "Get" , lets just call it as /entityRelationship. GET is indicated by the HTTP Method

Reply:
@OnkarVO7 any reason not to add this at database level?

Reply:
@harshach Main to add this at schema level was to make sure the performance remains consistent. We can extend database level too once this is out